docs/search/search-query-language/search-operators/num.md

```diff
@@ -4,7 +4,7 @@ title: num Search Operator
 sidebar_label: num
 ---
 
-The `num` operator converts a field to a floating point number. Using num in a query is useful for sorting results by number instead of alphabetically, which is the default.
+The `num` operator converts a field to a *double* value (64-bit IEEE 754 double-precision floating-point number), which is twice as accurate as a float value (32-bit IEEE 754 single-precision floating-point number). Using `num` in a query is useful for sorting results by number instead of alphabetically, which is the default.
```

Comment:
```suggestion
The `num` operator converts a field to a double value (64-bit IEEE 754 double-precision floating-point number), which is twice as accurate as a float value (32-bit IEEE 754 single-precision floating-point number). Using `num` in a query can be useful for sorting results by number instead of alphabetically, which is the default.
```